rotkehlchen/globaldb/upgrades/v8_v9.py

```diff
@@ -13,6 +13,12 @@ def _addressbook_schema_update(write_cursor: 'DBCursor') -> None:
     fix_address_book_duplications(write_cursor=write_cursor)
 
 
+@enter_exit_debug_log()
+def _add_uniswap_to_price_history_source_types(write_cursor: 'DBCursor') -> None:
+    """Add entries for Uniswap V2 and V3 to price_history_source_types table"""
+    write_cursor.execute("INSERT OR IGNORE INTO price_history_source_types(type, seq) VALUES ('G', 7), ('H', 8);")  # noqa: E501
```

Comment:
nitpick: why not do an executemany and use `?.?` and bindings? I did not know what you wrote actually works :smile: